### PR TITLE
Ui tweaks

### PR DESCRIPTION
**Pull request by jah2488**
These are the major changing points:
- Fixing student signup url
- Moving Assignments to top of page.
- Adding attendance card even if no day is set.
- Removing auto expanding js snippet. It was really broken.
- Changing the days calendar to use materialize and cards.
- Tweaking assignment columns
- Reformatting new day screen to be centered.
- Adding a button to 'current assignment'
- Made the submit button for submissions more noticeable.
- Removed the table layout in favor or ul/li
- Removed the icon/link to students and made the avatar and name a link instead.

I'm going to make a few gifs and add them as comments so the visual changes are pretty easy to see from here. Hoping to get this merged and deployed by EOL tomorrow.

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -54,6 +50,4 @@ var ready = function() {
 };
 
 
-jQuery(document).ready(ready);
-jQuery(document).on('page:load', ready);
-
```
Comment (jah2488) [inline app/assets/javascripts/application.js]: Didn't realize you could do this, yay!

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -32,11 +33,6 @@ var ready = function() {
                 Location.getCurrentPosition();
         }
 
-        jQuery('textarea').on('keyup', function (e) {
-                var elem = jQuery(this);
-                elem.css('height', elem[0].scrollHeight);
-        });
```
Comment (jah2488) [inline app/assets/javascripts/application.js]: This was always garbage, will add back when it actually works.

**Comment by jah2488** [comment]
## Tweaked Instructor Dashboard
screen shot 2015-10-12 at 10 57 51 pm

## Updated Days Index
screen shot 2015-10-12 at 11 07 11 pm


## Student Dashboard Changes
screen shot 2015-10-12 at 11 08 30 pm
screen shot 2015-10-12 at 11 08 46 pm

File: app/assets/javascripts/application.js (hunk 2 above)
Comment (kurtisnelson) [inline app/assets/javascripts/application.js]: Ah that's what was fighting materialize.

**Comment by kurtisnelson** [comment]
:+1: once travis is happy. More titleize bugs though: PhD breaks it.

**Comment by jah2488** [comment]
Ohh missed those. Will fix up the specs in the morning.

**Comment by jah2488** [comment]
come on travis :(

**Comment by jah2488** [comment]
Wooooo

**Pull request merged**